CONTRIBUTING.md

```diff
@@ -14,25 +14,17 @@ To setup your environment:
   brew update
   brew install pyenv
   ```
-1. Install Pyenv versions for the Tox Suite
+1. Install your favorite Python version (>= 3.8 please!)
   ```bash
-  pyenv install 3.5.8
-  pyenv install 3.6.10
-  pyenv install 3.7.7
-  pyenv install 3.8.2
-  pyenv install pypy3.6-7.3.1
+  pyenv install <YOUR_FAVORITE_VERSION>
   ```
 1. Install Poetry
   ```bash
-  curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+  pip install poetry
```

Comment:
I checked the doc today and seems installing with pip is still not recommended.. 😓 

https://python-poetry.org/docs/

Reply:
oh lol, thanks for double checking!!  guess i will have to reinstall my local python and use the recommended way.  will revert this change when i return to this PR later this week.